src/index.js

```diff
@@ -12,35 +12,28 @@
 // @license       2015
```

Comment:
nah then we'd have to renew our license. I say we stick to our 2015 license for now. @shanek21 @altodyte you agree?